saas/office365/security_alerts/o365-security-alerts.pt

```diff
@@ -0,0 +1,144 @@
+name "Office 365 Security Alerts"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will identify Security Alerts that have been raised in Office 365. Policy Managers can minimize the notifications by choosing to only be alerted by certain severity level(s). See the [README](https://github.com/rightscale/policy_templates/tree/master/fsm/office_365/security_alerts/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Danger doesn't like these links.   the github link should be changed to flexera and the docs should be an https.